app/domain/operations/hbx_enrollments/drop_enrollment_members.rb

```diff
@@ -29,17 +30,19 @@ def drop_enrollment_members(params)
         terminate_date = Date.strptime(params[:options]["termination_date_#{params[:hbx_enrollment].id}"], "%m/%d/%Y")
         dropped_enr_members = params[:options].select{|string| string.include?("terminate_member")}.values
         all_enr_members = params[:hbx_enrollment].hbx_enrollment_members
-        eligible_members = all_enr_members.reject{ |member| dropped_enr_members.include?(member.id.to_s) }
+        non_eligible_members = all_enr_members.select{ |member| dropped_enr_members.include?(member.id.to_s) }
+        new_effective_date = terminate_date + 1.day
+        optionals_params = params[:hbx_enrollment].is_ivl_by_kind? ? ivl_params(params[:hbx_enrollment], new_effective_date) : {}
```

Comment:
is_ivl_by_kind check not needed here

Reply:
The behavior is different for shop enrollment so specifically taken the ivl_kind as suggested in the meeting to develop the tool specific to Ivl. And left this line for future development.

Reply:
Lets make the change assuming we are supporting for IVL market. we can revisit this when we make change for SHOP later

Reply:
Removing ivl check and changing the method as suggested.